src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
-
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/Bloggy
+spring.datasource.username=Kuba
+spring.datasource.password=1234
```

Comment:
please remove login credentials from repo for your own sake

Reply:
Credentials removed. File added to .gitignore.
application.properties.example added to Git.

Reply:
good job!